test/validation/api/queue/queue.c

```diff
@@ -55,12 +55,18 @@ static void generate_name(char *name, uint32_t index)
 
 static int queue_suite_init(void)
 {
+	odp_pool_capability_t pool_capa;
 	odp_shm_t shm;
 	test_globals_t *globals;
 	odp_pool_param_t params;
 	int num_workers;
 	odp_cpumask_t mask;
 
+	memset(&pool_capa, 0, sizeof(odp_pool_capability_t));
```

Comment:
memset() before odp_pool_capability() is not needed (unless one is testing that odp_pool_capability() works, which is not the point of this test). So this could be removed (but see my other comment).

Btw, if the memset was needed, it would be better to use sizeof(pool_capa) to avoid the explicit type name that must be kept in sync with pool_capa definition.